package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "terra-kaiju-plugin",
+  "version": "0.0.0",
+  "description": "Common utils for kaiju plugins",
+  "main": "PluginUtils.js",
```

Comment:
Did you want main to be `index.js`?

Reply:
Good catch, i copied this project from my kaiju-plugin-utils project